spring4/pom.xml

```diff
@@ -87,19 +87,4 @@
             </plugin>
         </plugins>
     </build>
-
-    <profiles>
-        <profile>
-            <id>jdk9</id>
-            <activation>
-                <jdk>[1.9,</jdk>
```

Comment:
@qualidafial is this still needed?  It seems to break the jdk11 build (unused dependency `annotation-api`) and removing it doesn't seem to break the jdk9 travis build...

Reply:
I was kinda throwing stuff at the wall to get the jdk9 build to work. If removing it hurt the build, go for it.

Reply:
Also I think this range should be `[1.9,)` (with the trailing `)` paren)--I seem to remember a PR from Dropwizard about this causing a problem.

Reply:
In that case I'll just remove it.  We can always add it back in if we remember why it's needed.